File: C/INICIANTE/2057 Fuso Horário.c
#include <stdio.h>

void calcularChegada(int s, int t, int f);

int main() {
    int saida, tempoViagem, fusoHorario;
    scanf("%d %d %d", &saida, &tempoViagem, &fusoHorario);
    calcularChegada(saida, tempoViagem, fusoHorario);
    return 0;
}

void calcularChegada(int s, int t, int f)
{
    if ((s+t+f) >= 24)
        printf("%d\n", ((s+t+f) - 24));
    else if ((s+t+f) < 0)
        printf("%d\n", ((s+t+f) + 24));
    else
        printf("%d\n", (s+t+f));
}

File: Python/BEGINNER/2057 Fuso Horário.py
#SUBPROGRAMA
def calcularTempoDeChegada(s, t, f):
    if (s+t+f) >= 24:
        print("%d" % ((s+t+f) - 24))
    elif (s+t+f) < 0:
        print("%d" % ((s+t+f) + 24))
    else:
        print("%d" % (s+t+f))
    return None

#PROGRAMA PRINCIPAL
saida, tempoViagem, fusoHorario = map(int, input().split())
calcularTempoDeChegada(saida, tempoViagem, fusoHorario)
